testing/env_python_3_with_R_and_tensorflow.yml

```diff
@@ -35,4 +36,4 @@ dependencies:
   - r-irkernel
   - r-shiny
   - r-googlevis
-  - tensorflow=1.13
+  - tensorflow
```

Comment:
It's only used in submarine testing.

BTW, does the community consider removing submarine support? 

- submarine was listed at [here](https://cwiki.apache.org/confluence/display/ZEPPELIN/Interpreter+Maintenance) with no clear decision
- there is no functionality change / bugfix in submarine module since 2020
- submarine already provided the built [notebook](https://submarine.apache.org/docs/designDocs/notebook-implementation) in the new version, and submarine docs does not mention zeppelin

thus, I suppose we can remove it?

Reply:
cc @jongyoul @Reamer @xunliu

Reply:
I do not use submarine. As long as it doesn't cause any problems, we should keep it.

Reply:
@Reamer OK, thanks for your advice.